satellite/console/consoleweb/consoleql/query.go

```diff
@@ -216,6 +218,30 @@ func rootQuery(service *console.Service, mailService *mailservice.Service, types
 					return true, nil
 				},
 			},
+			UserPaymentMethods: &graphql.Field{
+				Type: graphql.NewList(types.paymentMethod),
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					methods, err := service.GetUserPaymentMethods(p.Context)
+					if err != nil {
+						return nil, err
+					}
+
+					var userPayments []console.UserPaymentMethodCombined
```

Comment:
Why not define the `UserPaymentMethodCombined` only in this package, if it's only used here?